Ansible/roles/marvin/tasks/main.yml

```diff
@@ -92,6 +92,16 @@
   tags:
     - marvin
 
+- name: Install pyvmomi
+  pip:
+    name: "{{ item }}"
+    executable: pip3
+    extra_args: '--upgrade'
+  with_items:
+  - pyvmomi==8.0.3.0.1
```

Comment:
there is a section in Ansible/roles/marvin/tasks/install_marvin_prereqs.yml
maybe move into the section ?

```
- name: Update pip3 and components
  pip:
    name: "{{ item }}"
    executable: pip3
    extra_args: '--upgrade'
  with_items:
  - pip==21.3.1
  - six
  - pyOpenSSL
  - wheel==0.34.2
  - kubernetes
  - '"requests[security]"'
  - pyasn1
  - wget
  - texttable
  - lxml
  - pycparser==2.13
  - paramiko
  - mysql-connector-python==8.0.29
  - netaddr==0.10.1
  tags:
    - marvin
    - marvin_install
```